tests/operators/test_agnostic_aggregate_check.py

```diff
@@ -33,253 +36,213 @@
 
 import astro.sql as aql
 from astro.sql.table import Table
-from tests.operators.utils import DEFAULT_SCHEMA
+from tests.operators.utils import DEFAULT_SCHEMA, get_dag, get_table_name, run_dag
 
 log = logging.getLogger(__name__)
 DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+CWD = pathlib.Path(__file__).parent
+
+DATABASES_LIST = ["postgres", "bigquery", "sqlite", "snowflake"]
+
+
+@pytest.fixture(scope="session")
+def tables(request):
```

Comment:
@utkarsharma2 my feeling is that you adopted a strategy different from what we've been using so far:
https://github.com/astro-projects/astro/blob/48b09347170afc0e644a7db131b7ba11d01dfa53/conftest.py#L62-L75

In other integration tests, we have a fixture that varies depending on the DB (passed using `request.params`) - and returns a single instance. In this case, you changed the approach to return a list of databases. What do you think if we were consistent?

It also feels that a significant part of the logic here (parameters specified by the database) is available in other places, and perhaps could be reused/refactored - so we have less code duplication:
https://github.com/astro-projects/astro/blob/48b09347170afc0e644a7db131b7ba11d01dfa53/tests/operators/utils.py#L17-L29